app/views/accounts/index.html.erb

```diff
@@ -28,7 +28,11 @@
                 <% @accounts.each do |account| %>
                   <tr>
                     <td><%= account.name %></td>
-                    <td><%= account.domain %></td>
+                    <td>
+                      <%= link_to account.domain, target: :_blank, rel: 'noopener' do %>
```

Comment:
Akkor is generálunk egy linket, ha nincs megadva domain. Szemmel jelenleg nem látszik, viszont ezt egy későbbi új stílus bevezetésével bármikor kibukhat, mint jövőbeli probléma.

![image](https://github.com/AppMaintainers/beloved-projects/assets/1900366/8eab0ee3-6a9d-4660-8280-1fdf6fd73084)

Kérlek csak akkor rakjunk ki egy linket, ha tényleg van tartalma